04.ls/ls.rb

```diff
@@ -57,4 +88,90 @@ def puts_table(entry_name_table, max_width)
   end
 end
 
+def create_entry_info_table
+  entry_info_table = entry_names.map do |entry_name|
+    entry_info = {}
+    stat = File::Stat.new(entry_name)
+    entry_info[:file_mode] = get_file_mode(stat, entry_name)
+    entry_info[:nlink] = stat.nlink.to_s
+    entry_info[:owner_name] = get_owner_name(stat)
+    entry_info[:group_name] = get_group_name(stat)
+    entry_info[:file_size] = stat.size.to_s
+    updated_day = File.mtime(entry_name)
+    entry_info[:updated_dates] = get_updated_dates(updated_day)
+    entry_info[:entry_name] = entry_name
```

Comment:
```suggestion
    entry_info[:name] = entry_name
```

entry_infoに格納される名前なのでentryは省略してもいいかも。

また、owner_name、file_sizeも単にowner、sizeだけでいいかもしれませんね。

Reply:
```entry_name→name,owner_name→owner, file_size→size```に修正しました。